jinahub/encoders/image/AudioCLIPImageEncoder/Dockerfile

```diff
@@ -1,16 +1,18 @@
-FROM jinaai/jina:2.0.10-perf
+FROM jinaai/jina:2-py37-perf
 
 # install git
-RUN apt-get -y update && apt-get install -y git \
+RUN apt-get -y update && apt-get install -y git wget \
     && rm -rf /var/lib/apt/lists/*
 
 # install requirements before copying the workspace
-COPY requirements.txt /requirements.txt
+COPY requirements.txt requirements.txt
 RUN pip install -r requirements.txt --no-cache-dir
 
 # setup the workspace
-COPY . /workspace
-WORKDIR /workspace
+COPY . /workdir
+WORKDIR /workdir
 
-ENV PYTHONPATH=/workspace
+RUN ./scripts/download_full.sh
```

Comment:
Should not be here - users should download the file before starting the executor. You will need to add `--volumes` directive in docker tests